### DISCUSSION TITLE
Integrazione Google Home - Home Assistant - Impossibile raggiungere [test]

### DISCUSSION THREAD

**Discussion by Teddydm71** [indomus/forum#3061] [HUB: Home Assistant]
Buonasera a tutti, 
utilizzo Home Assistant OS su un MiniPC dedicato,
Ho attivato il servizio duckdns.org e la registrazione funziona.
ho installato Caddy per il reverse proxy in https sulla porta 9000 e la gestione automatica del certificato Let's Encrypt e funziona 
Sito raggiungibile attraverso https://myaddress.duckdns.org:9000.
Ho attivato l'app su Google Actions e censito l'utente di servizio su Google Cloud.
Quando apro google home dal cellulare ed aggiungo la mia app, mi appare la pagina di login, inserisco i dati di accesso e google home prova il collegamento.
Dopo pochi secondi mi butta fuori con errore "Impossibile raggiungere [test] - My App. Riprova".
Ho provato a cercare l'errore su google e ci sono diversi casi, ma le cui soluzioni, solo di alcuni casi peraltro, sono abbastanza casuali non trovano un fondamento....
Avete idee, suggerimenti, approfondimenti che posso fare?
Grazie mille

**Comment by Assistnew** [comment]
Buongiorno, stesso problema anch'io. Ho provato a riconfigurare Google Actions, Google Cloud Platoform secondo la guida... ma niente. Appare l'errore "Impossibile raggiungere [test] - My App. Riprova". C'è qualche soluzione?
Grazie

**Reply by Teddydm71** [reply]
Ciao, tanto non risponderà nessuno... :-(
Io ho risolto attivando Nabu Casa... funziona benissimo e, in fondo, il costo dell'abbonamento è ragionevole :-) 75 euro per un anno... se li meritano tutti.

**Reply by snakuzzo** [reply]
E invece qualcuno risponderà.
Questa risposta non ha alcun senso e non è una soluzione a quanto indicato.

**Reply by Teddydm71** [reply]
Scusami ma davvero vuoi convincermi di una cosa che è palese? Il mio topic è del 30 aprile e nessuno ha risposto. Siccome Assistnew ha fatto un reply al mio topic mi è arrivata la notifica. Mi è sembrato corretto rispondergli e rappresentare la soluzione ottimale al problema che avevo io a suo tempo e che ora ha lui: integrare home assistant con google home. Buona giornata.
Edoardo.

**Reply by snakuzzo** [reply]
Non è una soluzione al problema, ma un modo diverso di fare una cosa.
Quanto alla tua velata polemica sulle mancate risposte, non è detto che uno abbia letto, avuto tempo, voglia, possibilità di risponderti.
Inoltre, con i dettagli "praticamente nulli" che hai dato è sostanzialmente impossibile darti una risposta.
Più di dirti di riverificare tutto il giro fatto per l'integrazione, non saprei.

Poi...dai un'occhiata [QUI](https://github.com/indomus/forum/discussions/3129#discussioncomment-10032155)

**Reply by snakuzzo** [reply]
Precisiamo inoltre, che questa è una community e non un centro assistenza

**Reply by Teddydm71** [reply]
Questioni di punti di vista rispetto a cosa consideri "problema". Il mio era integrare google home con home assistant. Ho provato la soluzione gratuita e non ha funzionato nonostante avessi verificato e rifatto 2 volte tutti i passaggi. Siccome volevo risolvere il problema ho attivato la prova gratuita di Nabu Casa e l'ho trovata assolutamente adeguata alle mie esigenze per velocità di implementazione e per i relativi costi. Mi spiace che la mia risposta sia stata interpretata come una polemica; ti assicuro che non voleva esserlo. Era una constatazione dovuta al fatto che probabilmente una soluzione non c'era; quindi nessuno ha risposto perchè nessuno aveva suggerimenti. Grazie per l'indicazione spero che possa essere utile ad Assistnew.
